modules/ROOT/pages/audit-logging.adoc

```diff
@@ -29,7 +31,18 @@ Each log entry has a set of properties that provides information about the activ
 * *Type*: The type of the object on which the action is performed, for example, `Organization`
 * *Action*: The action associated with the object, for example, `Create`
 * *Object*: The name of the Object, for example, `foo`
-* *User*: The User who performed the action, for example, `johndoe`
+* *User*: The User who performed the action, for example, `johndoe` 
++
+[NOTE]
+--
+Occasionally, there are `Anypoint Staff` log entries. This log entry is made when the audit action was performed by a user who does not belong to your master organization. +
```

Comment:
I think "master" organization can be confusing here. Since all users belong to the master org. Can we just say "your organization"?